modules/sdk-core/src/bitgo/baseCoin/baseCoin.ts

```diff
@@ -278,6 +278,13 @@ export abstract class BaseCoin implements IBaseCoin {
     return false;
   }
 
+  /**
+   *  Check whether a coin utxo-based
+   */
+  usesUnspentModel(): boolean {
+    return false;
+  }
```

Comment:
with the `typeof coin.signWithCustomSignerFunction === 'function'` we don't need this and are more flexible

Reply:
Done